From f6258900e4b987b3eba901061b11df63a33ccd42 Mon Sep 17 00:00:00 2001
From: Graham Moore <graham.moore@linux.intel.com>
Date: Fri, 16 Jun 2017 13:58:38 -0500
Subject: [PATCH 025/151] FogBugz #468942-2: Modify QSPI read_delay for
 Arria10.

commit  df9b9fb0e887dcdcefd33e401a4889a71f4db6ab from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

The default read delay for Arria10 Cadence QSPI should be
3 instead of 4 in order to recognize the QSPI device
that is shipped with the dev kit.

Signed-off-by: Graham Moore <graham.moore@linux.intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10_socdk_qspi.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/socfpga_arria10_socdk_qspi.dts b/arch/arm/boot/dts/socfpga_arria10_socdk_qspi.dts
index b4c0a76a4d1a..2b645642b935 100644
--- a/arch/arm/boot/dts/socfpga_arria10_socdk_qspi.dts
+++ b/arch/arm/boot/dts/socfpga_arria10_socdk_qspi.dts
@@ -19,7 +19,7 @@
 		m25p,fast-read;
 		cdns,page-size = <256>;
 		cdns,block-size = <16>;
-		cdns,read-delay = <4>;
+		cdns,read-delay = <3>;
 		cdns,tshsl-ns = <50>;
 		cdns,tsd2d-ns = <50>;
 		cdns,tchsh-ns = <4>;
-- 
2.26.1

